test/sunburst-label.html

```diff
@@ -112,6 +114,120 @@
+                                        {
+                                            name: 'Jackson',
+                                            value: 2
+                                        }
+                                    ]
+                                },
+                                {
+                                    name: 'Cousin Ben',
+                                    value: 4
+                                }
+                            ]
+                        },
+                        {
+                            name: 'Father',
+                            value: 10,
+                            children: [
+                                {
+                                    name: 'Me',
+                                    value: 5
+                                },
+                                {
+                                    name: 'Brother Peter',
+                                    value: 1
+                                }
+                            ]
+                        }
+                    ]
+                },
+                {
+                    name: 'Nancy',
+                    children: [
+                        {
+                            name: 'Uncle Nike',
+                            children: [
+                                {
+                                    name: 'Cousin Betty',
+                                    value: 1
+                                },
+                                {
+                                    name: 'Cousin Jenny',
+                                    value: 2
+                                }
+                            ]
+                        }
+                    ]
+                }
+
+            ];
+            option = {
+                series: {
+                    type: 'sunburst',
+                    startAngle: 0,
+                    // emphasis: {
+                    //     focus: 'ancestor'
+                    // },
+                    data: data,
+                    radius: [0, '90%'],
+                    label: {
+                        rotate: 'radial'
+                    }
+                }
+            };
+            var chart = testHelper.create(echarts, 'main1', {
+                title: [
+                    'Label rotation under different startAngle '
+                ],
+                option: option,
+                height: 600,
+                // buttons: [{text: 'btn-txt', onclick: function () {}}],
+                // recordCanvas: true,
+            });
+
+            var config = {
+                startAngle: 0
+            };
+
+            function update() {
+                chart.setOption({
+                    series: [{
+                        startAngle: config.startAngle
+                    }]
+                });
+            }
+
+            var gui = new dat.GUI();
+                gui.add(config, 'startAngle', 0, 360)
```

Comment:
As per https://github.com/apache/echarts/issues/18023#issuecomment-1337952610, it's better to also add `clockwise`.

Reply:
`clockwise` has added